### PR TITLE
Pillar support

### PR DESCRIPTION
**Pull request by krlmlr**
Requires hadley/colformat#38.

Test output: https://travis-ci.org/tidyverse/hms/jobs/271835256#L1906

### CODE REVIEW

File: R/colformat.R [hunk 1]
```diff
@@ -0,0 +1,44 @@
+cf_data.hms <- function(x, ...) {
```
Comment (hadley) [inline R/colformat.R]: I think you need a comment here along the lines of "dynamically exported; see `zzz.R`

Comment (krlmlr) [reply R/colformat.R]: Done.

**Comment by krlmlr** [comment]
Added a test for hms with 1000 hours.

**Comment by hadley** [comment]
Looks good and NA placement is much better.

**Pull request merged**